src/components/Header.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] KIndly note that it is not encouraged for you to disable linters in your application because linters ensure that you write bug-free and readable code. So kindly remove all the linters you disabled in all your app 😄